components/bl-rich-editor-component/styles/index.less

```diff
@@ -0,0 +1,245 @@
+@bl-customComponent-richEditor-borderStyle: solid;
+@bl-customComponent-richEditor-borderColor: #ccc;
+@bl-customComponent-richEditor-border: @bl-customComponent-richEditor-borderWidth @bl-customComponent-richEditor-borderStyle @bl-customComponent-richEditor-borderColor;
+
+.bl-customComponent-rich-editor {
+  display: flex;
+  flex-direction: column;
+  border: @bl-customComponent-richEditor-border;
+
+  #editor {
+    font-family: @bl-customComponent-richEditor-editor-fontFamily;
+    font-size: @bl-customComponent-richEditor-editor-fontSize;
+    border-width: @bl-customComponent-richEditor-editor-borderWidth;
+    height: @bl-customComponent-richEditor-editor-height;
+
+    pre {
+      background: @bl-customComponent-richEditor-disabledColor;
+      color: contrast(@bl-customComponent-richEditor-disabledColor);
+    }
+
+    .ql-editor.ql-blank::before {
+      color: @bl-customComponent-richEditor-editor-placeholder-color;
+    }
+  }
+
+  #toolbar-container {
+    border-color: @bl-customComponent-richEditor-borderColor;
+    border-style: @bl-customComponent-richEditor-borderStyle;
+    border-width: @bl-customComponent-richEditor-toolbar-borderWidth;
+
+    .ql-picker.ql-size .ql-picker-label:not([data-value])::before {
+      content: @bl-customComponent-richEditor-toolbar-defaultContent-fontSize;
+    }
+
+    .ql-picker.ql-font .ql-picker-label:not([data-value])::before {
+      content: @bl-customComponent-richEditor-toolbar-defaultContent-font;
+    }
+
+    .ql-stroke {
+      stroke: @bl-customComponent-richEditor-textColor;
+    }
+
+    .ql-fill {
+      fill: @bl-customComponent-richEditor-textColor;
+    }
+
+    .ql-picker {
+      color: @bl-customComponent-richEditor-textColor;
+
+      .ql-picker-options {
+        background-color: @bl-customComponent-richEditor-backgroundColor;
+      }
+
+      .ql-picker-item {
+        color: @bl-customComponent-richEditor-textColor;
+      }
+    }
+
+    .ql-undo, .ql-redo {
+      display: flex;
+      align-items: center;
+      color: @bl-customComponent-richEditor-textColor;
+    }
+
+    button:hover,
+    button:focus,
+    button.ql-active,
+    .ql-picker-label:hover,
+    .ql-picker-label.ql-active,
+    .ql-picker-item:hover,
+    .ql-picker-item.ql-selected {
+      color: @bl-customComponent-richEditor-button-color;
+    }
+
+    button:hover .ql-fill,
+    button:focus .ql-fill,
+    button.ql-active .ql-fill,
+    .ql-picker-label:hover .ql-fill,
+    .ql-picker-label.ql-active .ql-fill,
+    .ql-picker-item:hover .ql-fill,
+    .ql-picker-item.ql-selected .ql-fill {
+      fill: @bl-customComponent-richEditor-button-color;
+    }
+
+    button:hover .ql-stroke,
+    button:focus .ql-stroke,
+    button.ql-active .ql-stroke,
+    .ql-picker-label:hover .ql-stroke,
+    .ql-picker-label.ql-active .ql-stroke,
+    .ql-picker-item:hover .ql-stroke,
+    .ql-picker-item.ql-selected .ql-stroke {
+      stroke: @bl-customComponent-richEditor-button-color;
+    }
+  }
```

Comment:
For hover should be another color that default, isn't it?

Reply:
Yes, you are right.
By default it will be as a text color:
@bl-customComponent-richEditor-textColor: @appTextColor;
And by hover as a theme color:
@bl-customComponent-richEditor-button-color: @bl-customComponent-richEditor-themeColor;